src/Orchard.Cms.Web/Modules/Orchard.Contents/Controllers/AdminController.cs

```diff
@@ -447,7 +447,24 @@ public Task<IActionResult> EditPOST(int id, string returnUrl)
 
         private async Task<IActionResult> EditPOST(int id, string returnUrl, Func<ContentItem, Task> conditionallyPublish)
         {
-            var contentItem = await _contentManager.GetAsync(id, VersionOptions.DraftRequired);
+            var contentItem = await _contentManager.GetAsync(id, VersionOptions.Latest);
+
+            if (contentItem == null)
+            {
+                return NotFound();
+            }
+
+            ContentTypeDefinition typeDefinition = _contentDefinitionManager.GetTypeDefinition(contentItem.ContentType);
+            var draftRequired = (!contentItem.Published) || (typeDefinition.Settings.ToObject<ContentTypeSettings>().Draftable);
```

Comment:
I know there is a reason for doing this but don't remember exactly, let me a few minutes.